CMakeLists.txt

```diff
@@ -338,23 +332,25 @@ find_package(Threads)
 
 if(NOT CMAKE_BUILD_TYPE)
 	set(CMAKE_BUILD_TYPE "Release" CACHE STRING
-		"Build type (Release/Debug/RelWithDebInfo/MinSizeRe)" FORCE)
+		"Build type (Release/Debug/RelWithDebInfo/MinSizeRel)" FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
 
-if(CMAKE_BUILD_TYPE STREQUAL Debug)
-	add_definitions(-D_DEBUG -ggdb)
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+	add_definitions(-D_DEBUG)
+	check_and_add_flag(GGDB -ggdb)
```

Comment:
This raises a warning on MSVC, but doesn't error, right? So the check will pass, and then proceed to spam the build output.